packages/color-scale-block/src/helpers/resizeEvenly.ts

```diff
@@ -1,12 +1,43 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { ColorProps, ColorScaleBlockRef } from '../types';
+import { ColorProps, ColorScaleBlockRef, ResizeEvenlyOptions } from '../types';
 import { calculateDefaultColorWidth } from './calculateDefaultColorWidth';
 
-export const resizeEvenly = (itemList: ColorProps[], colorScaleBlockRef: ColorScaleBlockRef) => {
-    const defaultWidth = calculateDefaultColorWidth(itemList.length, colorScaleBlockRef);
+export const resizeEvenly = (
+    unfilteredList: ColorProps[],
+    colorScaleBlockRef: ColorScaleBlockRef,
+    options?: ResizeEvenlyOptions
+) => {
+    const skipResizedColors = options?.skipResizedColors ?? null;
 
-    return itemList.map((item) => {
-        return { ...item, width: defaultWidth, resized: false };
+    const colorsToIncludeInCalculation = unfilteredList.filter((color) => {
+        if (skipResizedColors && color.resized) {
+            return false;
+        }
+
+        return true;
+    });
```

Comment:
```suggestion
    const colorsToIncludeInCalculation = unfilteredList.filter((color) => {
                return (skipResizedColors && color.resized) ? false : true;
    });
```

Reply:
or directly something like this `!(skipResizedColors && color.resized)`?

Reply:
This code has been removed.